DaS-PC-MPChan/DSCM.vb

```diff
@@ -526,24 +526,34 @@ Public Class DSCM
                     tmpWorld = tmpFields(5)
                     tmpUpdMinute = TimeOfDay.TimeOfDay.Minutes
 
-                    If tmpSteamID.Length = 16 Then
-
-
-                        'Autoconnect to any nodes when Currnodes are under 4
-                        If Val(txtCurrNodes.Text) < 4 Then
-                            attemptConnSteamID(tmpSteamID)
+                    'Calculations for the IRC node-connect feature.
+                    Dim CurrNodesThreshold As Integer = 4
```

Comment:
I don't see any issues with the ideas behind this code but I won't be able to test it out for a bit.  Also, I have to look into how to accept part of a submission.  I'll get back to you later tonight, probably.